src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -118,6 +124,20 @@ public Integer convert(String value) {
         }
     }
 
+    @Option(names = "--depth", description = "Perform shallow validation in which each SHACL rule (shape) is only applied to [num] target nodes, or 50 nodes if [num] is omitted (requires -t).",
+            paramLabel = "num", arity = "0..1", converter = DepthConverter.class)
+    private int depth = DEFAULT_DEPTH;
+    private static class DepthConverter implements CommandLine.ITypeConverter<Integer> {
+        @Override
+        public Integer convert(String value) {
+            try {
+                return "".equals(value) ? MINIMUM_DEPTH : Integer.parseInt(value);
```

Comment:
I believe this is where you intend to set the value to the default if num was omitted. The descriptions say the default is 50, but MINIMUM_DEPTH is set to 1.

Reply:
This was poorly/erroneously copied from `--abort` where 3 is both the minimum and the default.  Setting `MINIMUM_DEPTH` to 50 is insufficient because it rejects anything less than 50 (e.g., `--depth=49`).  I need to separate the minimum, default, and detection of whether or not you specified anything.  Will do so in a future commit.

Reply:
resubmitting for request changes

Reply:
This should be fixed now.  Let me know if you see any other errors.